lib/active_record/connection_adapters/sqlserver/core_ext/calculations.rb

```diff
@@ -31,8 +32,8 @@ def build_count_subquery(relation, column_name, distinct)
 
 ActiveSupport.on_load(:active_record) do
   if ActiveRecord::VERSION::MAJOR == 5 &&
-     ActiveRecord::VERSION::MINOR == 1 &&
-     ActiveRecord::VERSION::TINY >= 4
+     ActiveRecord::VERSION::MINOR == 2 &&
+     ActiveRecord::VERSION::TINY >= 0
```

Comment:
We can probably remove this `if` altogether as gem requirement is now `'~> 5.2.0'`.